content/References/ACLPrimer.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "ACL Primer"
+weight: 9
+---
+
+{{< toc >}}
+
+[TrueNAS SCALE brings full ACL compatibility]({{< relref "PermissionsSCALE.md" >}}) between Windows and Linux with NFSv4 ACLs on ZFS and eases the challenges of integrating Unix servers in Windows environments.
+
+## ACL Overview
+
+In TrueNAS, ACLs specify which users or system processes (trustees) have access to datasets or shares. ACLs also determine what operations trustees may perform.
+
+Each entry in an ACL specifies a trustee and an operation. For example, if an SMB share has an ACL that contains: 
+
+```  
+Who: User  
+User: user1  
+Permissions: Full Control  
+  
+Who: User  
+User: user2  
+Permissions: Read  
+```  
+
+The ACL would give *user1* permission to view, edit, and create files and directories within the share, but only give *user2* viewing permissions.
+
+## NFSv4 in TrueNAS
+
+While the POSIX ACL type lets administrators assign read, write, and execute permissions to the chosen *Who*, the NFSv4 ACL type lets administrators fine-tune the read, modify (write), and traverse (execute) permissions so that they apply to certain file traits and actions. 
+
```

Comment:
>the NFSv4 ACL type lets administrators fine-tune the read, modify (write), and traverse (execute) permissions so that they apply to certain file traits and actions

I don't know what a file trait or action is. You probably need to explain this more clearly since I don't believe those are terms normally associated with files.

Reply:
How does this sound?

```
While the POSIX ACL type has basic read, write, and execute permissions, the NFSv4 ACL type lets administrators fine-tune advanced read, modify (write), and traverse (execute) permissions. 

For example, NFSv4 advanced permissions allow an administrator to set up a trustee that can read and write data, but not delete anything.
```